### ISSUE TITLE
[BUG] Glow Esp renders black outline 

### ISSUE THREAD

**Issue by ccetl** [CCBlueX/LiquidBounce#3268]
### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.6.0

### Operating System

Windows 11

### Minecraft Version

1.20.6

### Describe the bug

Enable any esp module with glow mode and there won't be any "glow" but a gradient between the selected color and black.

### Screenshots

![image](https://github.com/CCBlueX/LiquidBounce/assets/109693935/a42599fc-7554-4fd9-88c1-6fab4c16bf47)

**Comment by liquidsquid1** [comment]
This is normal as it uses the default Minecraft glow. There is no current bloom/glow in the LiquidBounce ESP settings.

**Comment by ccetl** [comment]
> This is normal as it uses the default Minecraft glow. There is no current bloom/glow in the LiquidBounce ESP settings.

There is a mode called "glow".
![image](https://github.com/CCBlueX/LiquidBounce/assets/109693935/e7983779-52c0-468a-8d20-1dd897aa7c56)

**Comment by liquidsquid1** [comment]
> > This is normal as it uses the default Minecraft glow. There is no current bloom/glow in the LiquidBounce ESP settings.
> 
> There is a mode called "glow". ![image](https://private-user-images.githubusercontent.com/109693935/341455154-e7983779-52c0-468a-8d20-1dd897aa7c56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4OTUxMDYsIm5iZiI6MTcxODg5NDgwNiwicGF0aCI6Ii8xMDk2OTM5MzUvMzQxNDU1MTU0LWU3OTgzNzc5LTUyYzAtNDY4YS04ZDIwLTFkZDg5N2FhN2M1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMFQxNDQ2NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMDE0MjlmNzU5ZmExZDMzNzU2M2I5NDhhYWE5YjcwMGQ0YWNiNWNiNjgwNTJiYjdjOWQxOTczZDk5N2ExMDEzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.PIJWTY5CMTF5AjIQplTbI_dHUREkC7048qt5rtM09iw)

and this is Minecraft glow, not Bloom.

**Comment by ccetl** [comment]
> > > This is normal as it uses the default Minecraft glow. There is no current bloom/glow in the LiquidBounce ESP settings.
> > 
> > 
> > There is a mode called "glow". ![image](https://private-user-images.githubusercontent.com/109693935/341455154-e7983779-52c0-468a-8d20-1dd897aa7c56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4OTUxMDYsIm5iZiI6MTcxODg5NDgwNiwicGF0aCI6Ii8xMDk2OTM5MzUvMzQxNDU1MTU0LWU3OTgzNzc5LTUyYzAtNDY4YS04ZDIwLTFkZDg5N2FhN2M1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMFQxNDQ2NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMDE0MjlmNzU5ZmExZDMzNzU2M2I5NDhhYWE5YjcwMGQ0YWNiNWNiNjgwNTJiYjdjOWQxOTczZDk5N2ExMDEzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.PIJWTY5CMTF5AjIQplTbI_dHUREkC7048qt5rtM09iw)
> 
> and this is Minecraft glow, not Bloom.

It's not what i understand under a "glow". Right now it's an outline with 2 colors. I'm aware that they use minecraft's effect.

**Comment by 1zun4** [comment]
Not a bug.

**Comment by ccetl** [comment]
> Not a bug.

Miss leading name. This is no glow.

**Comment by 1zun4** [comment]
Yes I know but we don't have anything better. Someone would have to write glow shaders for ESP.
